mappings/net/minecraft/entity/ai/goal/GoToOwnerAndPurrGoal.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS ami net/minecraft/entity/ai/goal/GoToOwnerAndPurrGoal
+	FIELD g cat Laqs;
```

Comment:
`owner`

Reply:
`actor`

Reply:
It extends MoveToTargetGoal, so `catActor`

Reply:
Just `cat` as rune suggested. Disregard my `actor` suggestions.